mobile/src/main/java/org/hawkular/client/android/activity/MainActivity.java

```diff
@@ -294,6 +300,21 @@ void openDrawerMenu() {
         drawer.openDrawer(GravityCompat.START);
     }
 
+    // --- OnCLick Event for clicking Fab --- Adds a new trigger ---------
+
+    @OnClick(R.id.fab)
+    void onClickFab(){
+
+        Bundle bundle = new Bundle();
+        bundle.putString("state", "From Home");
+        AlertSetupFragment alertSetupFragment = new AlertSetupFragment();
+        alertSetupFragment.setArguments(bundle);
+
+        adapter.addFragment(alertSetupFragment,"Setup Alert");
```

Comment:
I think this should be "Setup Trigger".
I wonder (but this does not need to be done in this PR) if it makes sense to externalise all those strings for easier translation/localization later on.